requirements.txt

```diff
@@ -3,4 +3,5 @@ kubernetes
 requests
 lightkube
 lightkube-models
-aiohttp
\ No newline at end of file
+aiohttp
+serialized_data_interface >= 0.4.0
```

Comment:
Should we start commenting inside `requirements.txt`?
```suggestion

# SDI library
# Used by: traefik-k8s
serialized_data_interface >= 0.4.0
```

Reply:
Since `serialized_data_interface` [it's going to be removed](https://github.com/canonical/traefik-k8s-operator/pull/28) soon from Traefik lib it's not necessary.